draft-irtf-cfrg-opaque.md

```diff
@@ -1646,6 +1646,62 @@ learn the client's password. This means that additional checks should be conside
 a production deployment of OPAQUE: for instance, ensuring that there is a
 server-authenticated channel over which OPAQUE registration and login is run.
 
+# API Considerations
+
+## Errors {#api-errors}
+
+This specification based on OPRF and 3DH only has a couple of fallible functions. For example,
+MACs exchanged during the AKE that might don't verify. Additionally, given the statically sized nature
```

Comment:
```suggestion
MACs exchanged during the AKE that might not verify. Additionally, given the statically sized nature
```
Not exactly sure, english isn't my mother tongue.

Reply:
Mine neither :) but you're correct !